exporter/exporterparser/README.md

```diff
@@ -2,12 +2,55 @@ A variety of exporters are available to the OpenCensus Service (both Agent and C
 
 ## Collector
 
+### Intelligent Sampling
+
+The Collector features intelligent (tail-based) sampling. In addition to different configuration
+options it features a variety of different policies.
+```yaml
+sampling:
+  mode: tail
+  # amount of time from seeing the first span in a trace until making the sampling decision
+  decision-wait: 10s
+  # maximum number of traces kept in the memory
+  num-traces: 10000
+  policies:
+    # user-defined policy name
+    my-rate-limiting:
+      # exporters the policy applies to
+      exporters:
+        - jaeger
+        - omnition
+      policy: rate-limiting
+      configuration:
+        spans-per-second: 1000
+#      policy: string-tag-filter
```

Comment:
I favor removing the comments, I can add some examples with all other policies and their configuration to other location.